triagebot.toml

```diff
@@ -29,3 +29,22 @@ cc = ["@grief8"]
 # @boterinas blocked - This indicates that a PR is blocked on something. 
 # @boterinas ready - This indicates that a PR is ready for review. 
 # @boterinas review or @boterinas reviewer are aliases for ready.
+
+[review-submitted]
+# This feature will automatically adjust the labels on a pull request when a reviewer sends a review with changes requested.
```

Comment:
This line explains `[review-submitted]`. Move it before the `[review-submitted]`.

Reply:
Thank you for the suggestions!